File: app/src/main/java/cn/tongdun/android/MyApplication.java
package cn.tongdun.android;


import android.app.Application;

import com.tencent.mmkv.MMKV;

import cn.tongdun.mobrisk.TDRisk;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        MMKV.initialize(this);

        boolean isAgree = MMKV.defaultMMKV().decodeBool("isAgreePrivacyAgreement");
        if (isAgree){
            TDRisk.init(this);
        }
    }
}

File: app/src/main/kotlin/cn/tongdun/android/MyApplication.kt
package cn.tongdun.android

import android.app.Application
import cn.tongdun.mobrisk.TDRisk.init
import com.tencent.mmkv.MMKV

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        MMKV.initialize(this)
        val isAgree = MMKV.defaultMMKV().decodeBool("isAgreePrivacyAgreement")
        if (isAgree) {
            init(this)
        }
    }
}